src/Twig/AppExtension.php

```diff
@@ -64,4 +71,11 @@ public function getLocales(): array
 
         return $this->locales;
     }
+
+    public function sanitize(string $unsafeHtmlContents): string
+    {
+        $htmlSanitizer = new HtmlSanitizer((new HtmlSanitizerConfig())->allowSafeElements());
```

Comment:
you should rather showcase using the service configured by FrameworkBundle (by injecting the HtmlSanitizerInterface)